packages/bot-skeleton/src/services/tradeEngine/trade/Balance.js

```diff
@@ -22,17 +22,10 @@ export default Engine =>
 
         // eslint-disable-next-line class-methods-use-this
         getBalance(type) {
-            const { scope } = this.store.getState();
             const { client } = DBotStore.instance;
             const balance = (client && client.balance) || 0;
-            let value = balance;
 
-            if (scope === 'BEFORE_PURCHASE') {
-                // Deduct trade amount in this scope for correct (🤦) value in balance-block
-                value = Number(balance) - this.tradeOptions.amount;
-            }
-
-            balance_string = getFormattedText(value, client.currency, false);
+            balance_string = getFormattedText(balance, client.currency, false);
```

Comment:
can you please explain before and after of this method?

Reply:
@sandeep-deriv read my answer below